recipe/meta.yaml

```diff
@@ -10,7 +11,11 @@ source:
   sha256: 34330e5ce276099e2e8950d9335db5a875689a4c6a56751ef3b1d8c537f887f6
 
 build:
-  number: 0
+  number: 1
+  run_exports:
+    # Symbols were only removed in update from 3 to 4
+    # https://abi-laboratory.pro/index.php?view=timeline&l=jemalloc
+    - {{ pin_subpackage(name, max_pin='x') }}
```

Comment:
`max_pin=None`? Looks like there has been no breakage since 4.0 and is it really necessary to pin to 5.x?

Reply:
That would work for me as well, I was just siding on the side of safety in case they break it from 5 to 6.

I am happy to change it to None.

Reply:
@xhochy, it is your call on what you want the pin to be. I don't know when there is going to be a 6.x release for jemalloc or what that would even look like.

Reply:
With this stability, `max_pin=None` sounds reasonable.

Reply:
@xhochy, the jemalloc update to include max_pin=None is ready to merge.